Automattic/wp-calypso
# Add e2e test for `wpcom_block_editor_post_publish_add_new_click` event

#### Changes proposed in this Pull Request

#55419 adds a `wpcom_block_editor_post_publish_add_new_click` tracks event to the block editor. Because it relies on a CSS selector it is likely to break in the future. This e2e test will notify us if that happens.

- Add support for the new "add new" post-publish button to the `GutenbergEditorComponent.publish()` test method
- Add `waitForNavigation` flag to `GutenbergEditorComponent.publish()` test method so that we can test for tracks events before page navigation completes
- Test the `wpcom_block_editor_post_publish_add_new_click` event is being sent in the page and post editor

#### Testing instructions



* Verify the `Tracks "wpcom_block_editor_post_publish_add_new_click" event` test is being run in CI
